sanity/squashfs.go

```diff
@@ -141,5 +141,9 @@ func checkSquashfsMount() error {
 		return fmt.Errorf("unexpected squashfs canary content: %q", content)
 	}
 
+	// lastly, verify that loop device is usable (which would not be caught by squashfs tests above)
```

Comment:
the comment needs to be expended to explain why we care about this device even if apparently mount  as above works?  when does the device gets into the picture?